src/BrowserFetcher.ts

```diff
@@ -321,10 +321,43 @@ function install(archivePath: string, folderPath: string): Promise<unknown> {
 }
 
 async function extractZip(zipPath: string, folderPath: string): Promise<void> {
-  try {
-    await extract(zipPath, {dir: folderPath});
-  } catch (error) {
-    return error;
+  const nodeVersion = process.version;
+
+  /* There is currently a bug with extract-zip and Node v14.0.0 that
+   * causes extractZip to silently fail:
+   * https://github.com/puppeteer/puppeteer/issues/5719
+   *
+   * Rather than silenty fail if the user is on Node 14 we instead
+   * detect that and throw an error directing the user to that bug. The
+   * rejection message below is surfaced to the user in the command
+   * line.
+   *
+   * The issue seems to be in streams never resolving so we wrap the
+   * call in a timeout and give it 100ms to resolve before deciding on
+   * an error.
+   *
+   * If the user is on Node < 14 we maintain the behaviour we had before
+   * this patch.
+   */
+  if (nodeVersion === 'v14.0.0') {
```

Comment:
`nodeVersion.startsWith('v14.')`?

Reply:
Or do you think this is something Node.js will ship a patch for?

Reply:
I was hopeful that Node will ship a patch for it (given https://github.com/nodejs/node/pull/32968).

But it could be safer to default to starts with `v14.` for now and then we can just ship a new release when the problem is solved, so let's do that 👍